DOCSP-41778 query size limit (#3355)
mongodb/docs-realm: source/includes/flex-sync-limitations.rst

Flexible Sync does not support querying on properties in Embedded Objects 
or links. For example, ``obj1.field == "foo"``.

Query Size Limit
~~~~~~~~~~~~~~~~

> The **size limit** for a subscription to a query from your client application is **256 kB**
> when using Device Sync. Exceeding this limit results in a

Reviewer: This still reads a little confusing to me. If I have correctly understood that the size limit is *per query in the subscription set*, not the total subscription set, we might want to reword a little just for clarity:

We don't need to say "when using Device Sync" because we're on the subscription query page where it's understood (hopefully!) that we're talking about Sync subscriptions. And we don't need to say "from your client application" because we're in the SDK client docs.

We should also update the wording on the C++ page once we're happy with this one.

```suggestion
The **size limit** for any given query in your subscription set is
**256 kB**. Exceeding this limit results in a
```
